### PR TITLE
docs(components-native): Update BottomSheet docs

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

Updating docs for BottomSheet, it's mobile only. The same error from ContentOverlay is happening here, when we press to open the BottomSheet, it crashes, so I added the disclaimer

I created a bug ticket for the issue above: https://jobber.atlassian.net/browse/JOB-75299

### Added

- Added BottomSheet docs

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/BottomSheet/BottomSheet.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### Heading
+
+Add a **heading** to the BottomSheet in cases where the user has initiated a
+preceding action which could result in a state change. In this case, the title
+should repeat the preceding action label.
+
+## Content guidelines
+
+BottomSheetOption follows the same content guidelines as the web
+[Menu.](https://atlantis.getjobber.com/?path=/docs/components-menu--menu#content-guidelines)
```
Comment (JeffMarvin) [inline docs/components/BottomSheet/BottomSheet.stories.mdx:53]: I think this link can now be a relative link.

**Review by JeffMarvin — CHANGES_REQUESTED** [review]
> The same error from ContentOverlay is happening here, when we press to open the BottomSheet, it crashes, so I added the disclaimer

Is there a bug ticket in JIRA for fixing this? (Or what is our plan for handling these?)

I just had one comment about a relative link.

File: docs/components/BottomSheet/BottomSheet.stories.mdx (hunk 1 above)
Comment (rodrigoeidelvein) [reply docs/components/BottomSheet/BottomSheet.stories.mdx]: Nice catch!

**Comment by rodrigoeidelvein** [comment]
@JeffMarvin I created ticket to investigate BottomSheet and ContentOverlay rendering issues. Thanks for the suggestion

**Review by JeffMarvin — APPROVED** [review]
Nice work!

**Pull request merged**